sage_modabvar/abvar.py

```diff
@@ -2022,6 +2054,44 @@ def _ambient_hecke_matrix_on_modular_symbols(self, n):
         self.__ambient_hecke_matrix_on_modular_symbols[n] = T
         return T
 
+    def frobenius_polynomial(self, p):
+        """
+        Computes the frobenius polynomial at `p`. If this abelian variety is
+        not simple, a ValueError will be raised.
```

Comment:
"If this abelian variety is not simple, a ValueError will be raised." -- don't use the passive voice.   How about "... not simple, raises a ValueError."